packages/builder/src/pages/builder/portal/users/users/index.svelte

```diff
@@ -65,10 +68,12 @@
     { column: "role", component: RoleTableRenderer },
   ]
   let userData = []
+  let invitesLoaded = false
+  let pendingInvites = []
+  let parsedInvites = []
```

Comment:
Most of the following changes are simply restructuring the file to be more consistent with our usual structure and more readable.